validate.js

```diff
@@ -32,6 +32,7 @@ export default class Validate extends Validates {
     super(props);
 
     this.state = {
+      validate: undef,  // callback function to be called
```

Comment:
Why does this need to be copied to `state`?

Reply:
In the original code, we were comparing `this.props.validate === nextProps.validate` to determine if we should revalidate. Since we no longer have access to the previous props in `getDerivedStateFromProps`, I thought this would be the way around that. I also considered `componentDidUpdate` as a potential alternative, however that doesn't get executed on the initial mount.